.pyenchant_pylint_custom_dict.txt

```diff
@@ -325,6 +325,7 @@ typechecking
 typeddict
 typehint
 typeshed
+typoes
```

Comment:
Don't blame me, blame https://www.merriam-webster.com/dictionary/typos
```suggestion
```

Reply:
I've been doing this mistakes for years. Years 😄